client/src/modules/CrossCheckPairs/pages/CrossCheckPairs/CrossCheckPairs.tsx

```diff
@@ -0,0 +1,189 @@
+          courseId,
+          pagination.pageSize as IPaginationInfo['pageSize'],
+          pagination.current as IPaginationInfo['current'],
+          orderBy.field,
+          orderBy.order,
+          filters.checker?.toString(),
+          filters.student?.toString(),
+          filters.url?.toString(),
+          filters.task?.toString(),
+        );
+        setCrossCheckList({
+          content: data.items,
+          pagination: data.pagination,
+          orderBy: {
+            field: orderBy.field,
+            order: orderBy.order,
+          },
+        });
+      } finally {
+        setLoading(false);
+      }
+    },
+    [crossCheckList.content],
+  );
+
+  useEffect(() => {
+    loadInitialData();
+  }, []);
+
+  const handleViewComment = ({ historicalScores, checker, messages }: CrossCheckPairDto) => {
+    modal.info({
+      width: 1020,
+      title: `Comment from ${checker.githubId}`,
+      content: historicalScores.map((historicalScore, index) => (
+        <Space direction="vertical" key={historicalScore.dateTime} style={{ width: '100%' }}>
+          <Comment
+            content={
+              <>
+                {dateTimeRenderer(historicalScore.dateTime)}
+                <PreparedComment text={historicalScore.comment}></PreparedComment>
+                {index === 0 &&
+                  messages.length > 0 &&
+                  messages.map(message => (
+                    <Message
+                      key={message.timestamp}
+                      message={message}
+                      currentRole={CrossCheckMessageDtoRoleEnum.Student}
+                      settings={{ areContactsVisible: true }}
+                    />
+                  ))}
+              </>
+            }
+          />
+          {historicalScore.criteria?.length ? (
+            <Collapse>
+              <Collapse.Panel key={historicalScore.dateTime} header="Detailed feedback">
+                <CrossCheckCriteria criteria={historicalScore.criteria} />
+              </Collapse.Panel>
+            </Collapse>
+          ) : null}
+        </Space>
+      )),
+    });
+  };
+
+  return (
+    <AdminPageLayout
+      session={props.session}
+      loading={loading}
+      title="Cross-Check"
+      courseName={props.course.name}
+      courses={[props.course]}
+    >
+      {contextHolder}
+      <BadReviewControllers courseTasks={courseTasks} courseId={props.course?.id} />
+      <CrossCheckPairsTable
+        loaded={loaded}
+        crossCheckPairs={crossCheckList.content}
+        pagination={crossCheckList.pagination}
+        onChange={getCourseScore}
+        viewComment={handleViewComment}
+      />
+      <style jsx>{styles}</style>
+    </AdminPageLayout>
+  );
+}
+
+const styles = css`
+  :global(.rs-table-row-disabled) {
+    opacity: 0.25;
+  }
+
+  :global(.table-score td, .table-score th) {
+    padding: 0 5px !important;
+    font-size: 11px;
+  }
+
+  :global(.table-score td a) {
```

Comment:
you won't like it, but consider adding scoping to css

Reply:
fixed